run.sh

```diff
@@ -45,21 +45,30 @@ fi
 # Set group users to specified group id (non unique)
 groupmod -o -g "${PGID}" unifi-video &>/dev/null
 
+# Create logs directory
+mkdir -p /var/lib/unifi-video/logs
+
 # check for presence of perms file, if it exists then skip setting
 # permissions, otherwise recursively set on volume mappings for host
 if [[ ! -f "/var/lib/unifi-video/perms.txt" ]]; then
   echo "[info] Setting permissions recursively on volume mappings..." | ts '%Y-%m-%d %H:%M:%.S'
 
-  volumes=( "/var/lib/unifi-video" "/var/log/unifi-video" )
+  volumes=( "/var/lib/unifi-video" "/var/lib/unifi-video/data/videos" )
```

Comment:
Ok, I had to go look at the actual container layout on this. It's confusing because of how many fricking symlinks they apparently use...

/USR/lib/unifi-video/data is a SYMLINK to /var/lib/unifi-video... I assume at some point the "but it's variable data, it belongs in var", "fine I'll fucking move it and symlink it to shut you up" conversation occurred...

So... technically speaking... the only volume you actually need is /var/lib/unifi-video. I'm not sure why the docs here indicate you need to make volumes for /usr/lib/unifi-video/data/videos... because in practice that would just point to the /var/lib/unifi-video/videos volume mount already.

I think the README docs are misleading and suggesting a volume that is not necessary.